pb/protos/profile.proto

```diff
@@ -56,4 +49,20 @@ message Profile {
         string large    = 4;
         string original = 5;
     }
+
+    message Stats {
+        uint32 followerCount  = 1;
+        uint32 followingCount = 2;
+        uint32 listingCount   = 3;
+        uint32 avgRating      = 4;
```

Comment:
I'm bike shedding here, but abbreviations are generally avoided in other place. I suggest `avgRating` -> `averageRating`